hedera-mirror-importer/src/main/resources/application.yml

```
@@ -13,6 +13,7 @@ hedera:
         restUsername: mirror_api
         schema: public
         username: mirror_node
+        connection-init-sql: ${spring.datasource.hikari.connection-init-sql}
```

Use camel case and this is the opposite dependency as we have for the others. We should invert so hikari uses db properties. Also, we can probably drop separate tempTableBufferSize property

```suggestion
        connectionInitSql: set temp_buffers='256MB'; set timezone TO 'UTC';
```